lib/travis/build/appliances/debug_tools.rb

```diff
@@ -26,6 +26,17 @@ def apply_enabled
                 sh.cmd "wget -q -O tmate.tar.xz #{static_build_linux_url}", echo: false, retry: true
                 sh.cmd "tar --strip-components=1 -xf tmate.tar.xz", echo: false
               end
+              sh.elif "$(uname) = 'FreeBSD'" do
+                sh.cmd 'sudo pkg install -y libevent msgpack libssh', echo: true
```

Comment:
Note that `sudo` is not available by default on BSDs; there are a few places in the codebase here where that's explicitly worked around by using `su`.